src/test/compile-fail/associated-const-type-parameter-arrays.rs

```diff
@@ -11,16 +11,23 @@
 #![feature(associated_consts)]
 
 pub trait Foo {
-    const MIN: i32;
+    const Y: usize;
+}
+
+struct Abc;
+impl Foo for Abc {
+    const Y: usize = 8;
+}
 
-    fn get_min() -> i32 {
-        Self::MIN //~ ERROR E0329
-    }
+struct Def;
+impl Foo for Def {
+    const Y: usize = 33;
 }
 
-fn get_min<T: Foo>() -> i32 {
-    T::MIN; //~ ERROR E0329
-    <T as Foo>::MIN //~ ERROR E0329
+pub fn test<A: Foo, B: Foo>() {
+    let _array: [u32; <A as Foo>::Y] = //~ error: the parameter type
+        [4; <A as Foo>::Y];
```

Comment:
Does the previous error prevent the repeat expr from emitting an error? If so, this probably needs a second test